tests/integration/defs/test_e2e.py

```diff
@@ -2335,12 +2335,19 @@ def test_ptp_quickstart_advanced_mixed_precision(llm_root, llm_venv):
                  "gemma/gemma-3-27b-it",
                  marks=(skip_post_blackwell,
                         pytest.mark.skip_less_device_memory(80000))),
+    pytest.param(
+        "Nano-v2-VLM",
+        "Nano-v2-VLM",
+        marks=pytest.mark.skip(reason="Nano V2 VLM ckpt is not released yet.")),
 ])
 def test_ptp_quickstart_multimodal(llm_root, llm_venv, model_name, model_path,
                                    modality, use_cuda_graph):
     # NOTE: individual tests need to be enabled in
     # tests/integration/test_lists/qa/examples_test_list.txt
 
+    if model_name == "Nano-v2-VLM" and modality == "video":
+        pytest.skip("Nano V2 VLM cannot support video modality.")
```

Comment:
why are we not supporting video in this PR?

Reply:
You can probably use @danielafrimi's [vLLM implementation](https://github.com/vllm-project/vllm/pull/23644) as reference